tests/test_flask.py

```diff
@@ -373,6 +373,21 @@ def test_ical_all(self):
         self.assertEqual(output.data.count('BEGIN:VEVENT'), 15)
         self.assertEqual(output.data.count('END:VEVENT'), 15)
 
+    def test_ical_meeting(self):
+        """ Test the ical_calendar_meeting function. """
+        self.__setup_db()
+
+        output = self.app.get('/ical/calendar/meeting/2/')
+        self.assertEqual(output.status_code, 200)
+        self.assertTrue('BEGIN:VCALENDAR' in output.data)
+        self.assertTrue('SUMMARY:test-meeting2' in output.data)
+        self.assertTrue(
+            'DESCRIPTION:This is another test meeting'
+            in output.data)
+        self.assertTrue('ORGANIZER:pingou' in output.data)
+        self.assertEqual(output.data.count('BEGIN:VEVENT'), 1)
+        self.assertEqual(output.data.count('END:VEVENT'), 1)
```

Comment:
@pypingou makes sense. However, I'd suggest the following approach:
1. write tests to assert ical objects is generated properly.
2. We assume that vobject lib is tested and ical file is properly generated from ical object.
3. We remove so many tests from the view level, just check if the url is returning a 200 response or not.

I believe, this will be a more unittest-ish approach.